kubernetes/ansible/roles/sunbird-monitoring/templates/alertrules.yaml

```diff
@@ -1,19 +1,20 @@
 kafka_topic_prefix: "{{ env_name }}"
 app_es_server_count: "{{ groups['es'] | length }}"
 log_es_server_count: "{{ groups['log-es-2'] | length }}"
+cassandra_server_count: "{{ groups['cassandra'] | length }}"
 log_es_threshold: 30
 secor_job_count: "{{ secor_job_count | d(13) }}"
 
-kafka_events_deviceprofile_backup_threshold: "{{ kafka_events_deviceprofile_backup_threshold }}"
-kafka_events_channel_backup_threshold: "{{ kafka_events_channel_backup_threshold }}"
-kafka_telemetry_derived_backup_threshold: "{{ kafka_telemetry_derived_backup_threshold }}"
-kafka_telemetry_failed_backup_threshold: "{{ kafka_telemetry_failed_backup_threshold }}"
-kafka_telemetry_ingestion_backup_threshold: "{{ kafka_telemetry_ingestion_backup_threshold }}"
-kafka_graph_events_backup_threshold: "{{ kafka_graph_events_backup_threshold }}"
-kafka_telemetry_raw_backup_threshold: "{{ kafka_telemetry_raw_backup_threshold }}"
-kafka_telemetry_unique_backup_threshold: "{{ kafka_telemetry_unique_backup_threshold }}"
-kafka_learning_failed_events_backup: "{{ kafka_learning_failed_events_backup }}"
-kafka_telemetry_denorm_backup: "{{ kafka_telemetry_denorm_backup }}"
-kafka_telemetry_pipeline_metrics_backup: "{{ kafka_telemetry_pipeline_metrics_backup }}"
-kafka_telemetry_extractor_failed_backup: "{{ kafka_telemetry_extractor_failed_backup }}"
-kafka_telemetry_assess_backup: "{{ kafka_telemetry_assess_backup }}"
+kafka_events_deviceprofile_backup_threshold: "{{ kafka_events_deviceprofile_backup_threshold | d(1000) }}"
```

Comment:
You can remove the d(1000) as we have these in defaults. Else please remove them from defaults.